next/app/events/calendar/page.tsx

```diff
@@ -0,0 +1,22 @@
+import Calendar from "./Calendar";
+
```

Comment:
Let's add metadata to this page as well. The metadata object is used by Next.js for search engine optimization. It could look something like this:
```js
import Calendar from ".Calendar";

export const metadata: Metadata = {
  title: "Events Calendar",
  description:
    "The Society of Software Engineers hosts a wide variety of weekly events, ranging from talks and company visits to committee meetings and large seasonal parties.",
};

export default function EventsCalendar() {
...
```

Reply:
Otherwise, this page is exactly what I had in mind for this MVP feature, with the added bonus of optimizing the display based on screen size. Phenomenal job!

Reply:
Done! Please let me know if the new push is ok

Reply:
Looks perfect! Approving now...